src/main/java/racingcar/domain/RacingGame.java

```diff
@@ -0,0 +1,59 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+
+    private static final String MUST_BE_INTEGER = "[ERROR] 시도 횟수는 숫자만 허용됩니다.";
+    private static final String MUST_BE_POSITIVE = "[ERROR] 1미만의 시도횟수는 허용되지 않습니다.";
+    private final String SIGN_OF_CAR_NAME = ",";
```

Comment:
static이 빠졌네요 :)

```suggestion
    private static final String SIGN_OF_CAR_NAME = ",";
```
---

그리고 사용자 입력을 분리하는 것이 RacingGame 도메인의 책임인지 한번 고민해보시면 좋을 것 같아요.

Reply:
사용자에 입력에 관한 처리이니 view에 있는 것이 맞는 것 같습니다. 수정했습니다.